cloudvolume/datasource/graphene/metadata.py

```diff
@@ -7,20 +7,26 @@
 
 from ... import exceptions
 from ... import paths
+from ...secrets import chunkedgraph_credentials
 from ..precomputed import PrecomputedMetadata
 
 class GrapheneMetadata(PrecomputedMetadata):
   def __init__(self, cloudpath, use_https=False, *args, **kwargs):
```

Comment:
The GrapheneMetadata class should also support providing the token directly (including blanking it). It should by default use the saved value. This will make it easier to support remixing components.

Reply:
done